awswrangler/athena/_utils.py

```diff
@@ -330,6 +331,7 @@ def get_query_columns_types(query_execution_id: str, boto3_session: Optional[bot
     )
 
 
+@Deprecated
```

Comment:
Why do we need to mark it as deprecated when are simply switching to clients instead of resources?

Reply:
It's not related - we already deprecated this method in 3.0 so I thought if we will still have a couple 2.x releases it would be good to warn customers about this

Reply:
Un-deprecated. Confused #2043 - looks like we didn't deprecate the method overall, just removed default bucket creation